Index: scripts.cfg
===================================================================
--- scripts.cfg	(revision 404)
+++ scripts.cfg	(working copy)
@@ -208,6 +208,7 @@
 @ server module rat_grenade             # Barterground hunters grenades
 @ server module q_hunter                # Barterground hunters quests
 @ server module critter_item_movement   # Critter item movement
+@ server module box_with_a_surprise     # boxes with random loot
 
 # Bind reserved functions
 @ server bind init                         main
