Index: FOTEXT.MSG
===================================================================
--- FOTEXT.MSG	(revision 404)
+++ FOTEXT.MSG	(working copy)
@@ -206,6 +206,14 @@
 Ла-ла-ла-ла ла-ла-лэй ла-ла-ли-ла-ла!}
 
 #
+# Surprise boxes
+#
+
+{700}{}{No loot... Sad..}
+{700}{}{Case was filled with earth, apparently someone decided to make a joke. }
+{701}{}{Cheers! Here something is!}
+
+#
 # Den store house
 #
 
